internal/httpserve/handlers/invite_test.go

```diff
@@ -58,6 +58,7 @@ func (suite *HandlerTestSuite) TestOrgInviteAcceptHandler() {
 	require.NoError(t, err)
 
 	recipientCtx := auth.NewTestContextWithOrgID(recipient.ID, userSetting.Edges.DefaultOrg.ID)
+	suite.enableModules(recipientCtx, recipient.ID, userSetting.Edges.DefaultOrg.ID, models.AllOrgModules)
```

Comment:
This should only require the base module
```suggestion
	suite.enableModules(recipientCtx, recipient.ID, userSetting.Edges.DefaultOrg.ID, models.CatalogBaseModule)
```